tests/test_cases/test_client/test_client_methods.py

```diff
@@ -441,7 +441,68 @@ def test_create_from_dict_array_collection_with_new_options(
 
         assert col_dict == new_col_dict
 
+    @pytest.mark.parametrize(
+        ("splitter", "expected"),
+        [
+            ({"vgrid": [1, 3]}, (3, 1)),
+            ({"arrays_shape": [1, 3]}, (3, 1)),
+            ({"vgrid": [3, 1]}, (1, 3)),
+            ({"arrays_shape": [3, 1]}, (1, 3)),
+        ],
+    )
     def test_create_from_dict_varray_collection(
+        self, client: Client, varray_collection: Collection, splitter: dict, expected: tuple
+    ):
+        """Check if creation from dict is correct."""
+        schema = {
```

Comment:
May be this schema should be moved to params?

Reply:
I think, it's better to have it as close to the test as possible, as it is not used anywhere else.